pyproject.toml

```diff
@@ -1,16 +1,19 @@
 [tool.poetry]
 name = "plaster_yaml"
-version = "0.1.2"
+version = "0.2.0"
 description = "A plaster plugin to configure pyramid app with Yaml"
 readme = "README.md"
 authors = ["Guillaume Gauvrit <guillaume@gauvr.it>"]
 include = ["CHANGELOG.md"]
 license = "BSD-derived"
 
 [tool.poetry.dependencies]
+# See: https://pypi.org/project/backports.entry-points-selectable
+importlib_metadata = { version = ">=3.6", python = "<3.10" }
```

Comment:
I was waiting to drop the python 3.7 support to remove pkg_resources usage.

Is it ok for you to drop python 3.7 support ?

importlib.metadata is avaible in python 3.8
so updating the ligne 13 with  the code below should be enough.

```
python = "^3.8"
```